File: back/BoostedLife/src/main/java/com/boostedlife/dto/GeneralDTO.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.boostedlife.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 *
 * @author jhonfre
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class GeneralDTO {

    private String nombre;
    private int dineroBolsillo;
    private int dineroBanco;
    private String trabajo;
    private String nombreCompleto;
    private String telefono;

}

File: front/boostedlife/src/app/model/GeneralDTO.ts
import { OwnedVehiclesDTO } from "./OwnedVehiclesDTO";
import { OwnedPropertiesDTO } from "./OwnedPropertiesDTO";

export class GeneralDTO{
    public nombre: string;
    public dineroBolsillo: number;
    public dineroBanco: number;
    public trabajo:string;
    public nombreCompleto:string;
    public telefono:string;
    public steamId:string;
    public vehicles: Array<OwnedVehiclesDTO> ;
    public propiertes:Array<OwnedPropertiesDTO> ;
}